### PR TITLE
Now POE graph accepts the bar chosen for the user

### PR DESCRIPTION
**Pull request by xchourio**

### CODE REVIEW

File: src/cptextras/dlcolors.py [hunk 1]
```diff
@@ -75,12 +75,18 @@ def prepare_canvas(tailoring=None,predictand='PRCP',type=None):
...
+                    barcolor=cmaps[user_color]
             elif any(x in predictand for x in ['TMAX','TMIN','TMEAN','TMED']):
                 vmin= 0
                 vmax= 1
                 mark='black'
-                barcolor=cmaps['DL_TEMP_POE_COLORMAP']
+                if user_color == None:
+                    barcolor=cmaps['DL_TEMP_POE_COLORMAP']
+                else:
+                    barcolor=cmaps[user_color]
```
Comment (aaron-kaplan) [inline src/cptextras/dlcolors.py:89]: You've only enabled `user_color` for `tailoring == 'POE'` and values of `predictand` that contain `TMAX`, `TMIN`, `TMEAN`, OR `TMED`. If `user_color` is provided, shouldn't we respect it no matter what the values of `tailoring` and  `predictand` are?

Comment (xchourio) [reply src/cptextras/dlcolors.py:89]: The modification also includes the PRCP predictand (When the tailoring is POE)... I can generalize the user_color option for the other tailorings, I understood that customize option only would be available for POE ..... 

Green light for implementing that for all options?

File: src/cptextras/dlcolors.py [hunk 2]
```diff
@@ -33,8 +33,8 @@ def make_cmap(colors, name, reverse=True):
 def colormap_info():
     ret = "Colormaps starting with 'cpt.' come from\nthe International Research Institute\nfor Climate & Society's Climate Predictability Tool\n\nColormaps starting with 'pycpt.' come from PyCPT.\n\nColormaps starting with 'cmc.' come\nfrom cmcrameri, all credit to Fabio Crameri\nsee their message:\n\n" + cmc.__doc__ + '\nAll other colormaps come from matplotlib\nPlease Cite Colormaps accordingly!'
     print(ret)
-
-def prepare_canvas(tailoring=None,predictand='PRCP',type=None):
+#if user_color != None:
```
Comment (aaron-kaplan) [inline src/cptextras/dlcolors.py:36]: Delete comment

**Review by aaron-kaplan — APPROVED** [review]
Looks good. I will test and publish next week.

**Pull request merged**